src/components/Metrics/graphUtils.ts

```diff
@@ -0,0 +1,265 @@
+import { CitationsHistogramType, ReadsHistogramType } from '@api';
+import { BasicStatsKey, CitationsHistogramKey, CitationsStatsKey, ReadsHistogramKey } from '@api/lib/metrics/types';
+import { ICitationsTableData, IReadsTableData } from './types';
+
+export interface IGraphData {
+  key: string;
+  values: IPair[];
+}
+
+export interface IPair {
+  x: string;
+  y: number;
+}
+export interface ICitationTableInput {
+  refereed: {
+    [CitationsStatsKey.NORMALIZED_NUMBER_OF_CITATIONS]: number;
```

Comment:
You should be able to do:
```
refereed: {
  [key in CitationsStatsKey]: number;
}
```
Unless I'm missing this is using any different keys?